demos/toolbar/default-toolbar.html

```diff
@@ -50,5 +51,6 @@
         Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean ultricies mi vitae est. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Vestibulum tortor quam, feugiat vitae, ultricies eget, tempor sit amet, ante. Donec eu libero sit amet quam egestas semper. Aenean ultricies mi vitae est.
       </p>
     </main>
+    <script src="/assets/common.js" async></script>
```

Comment:
Nit: Can we add a blank line before `<script>`? Ditto for other files.

I find it more readable to have the scripts separated from the visible elements with a little whitespace.